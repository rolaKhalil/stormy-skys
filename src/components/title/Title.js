import React from 'react'

const Title = () => (
    <div>
        <h1 className="title-container__title">Weather Finder</h1> 
        <h3 className="title-container__subtitle">What's the weather like today?</h3> 
    </div> 
)

export default Title