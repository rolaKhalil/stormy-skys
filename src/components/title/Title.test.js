import React from 'react'
import { shallow, configure } from 'enzyme'
import Title from './Title';
import renderer from 'react-test-renderer'

describe('<Title />', () => {
    it('renders without crashing', () => {
        const tree = renderer.create(
            <Title />
        )
        expect(tree.toJSON()).toMatchSnapshot()
    });
})