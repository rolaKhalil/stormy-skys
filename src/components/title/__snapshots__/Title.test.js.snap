// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders without crashing 1`] = `
<div>
  <h1
    className="title-container__title"
  >
    Weather Finder
  </h1>
  <h3
    className="title-container__subtitle"
  >
    What's the weather like today?
  </h3>
</div>
`;
